go/flags/endtoend/vttablet.txt

```diff
@@ -264,6 +264,7 @@ Flags:
       --query-log-stream-handler string                                  URL handler for streaming queries log (default "/debug/querylog")
       --querylog-filter-tag string                                       string that must be present in the query for it to be logged; if using a value as the tag, you need to disable query normalization
       --querylog-format string                                           format for query logs ("text" or "json") (default "text")
+      --querylog-mode string                                             Mode for logging queries. 'error' will only log queries that return an error. Otherwise all queries will be logged.
```

Comment:
What are the allowed values? What is the default value? I can infer this from code, but we should document it in the help text, similar to `querylog-format` (just above this line).

Reply:
currently there is no other value than `error`.

Reply:
If you have a better help message, I can put that in

Reply:
let's default it to "all"

Reply:
done